src/components/subtitlesync/subtitlesync.js

```diff
@@ -145,7 +156,9 @@ define(['playbackManager', 'layoutManager', 'text!./subtitlesync.template.html',
                         // if there is an external subtitle stream enabled
                         playbackManager.canHandleOffsetOnCurrentSubtitle(player)) {
                         // if no subtitle offset is defined
-                        if (!playbackManager.getPlayerSubtitleOffset(player)) {
+                        if (!(playbackManager.getPlayerSubtitleOffset(player) ||
+                        // or being defined (element has focus)
+                        subtitleSyncTextField.hasFocus)) {
```

Comment:
seems like this can be on a single line

Reply:
I made a line break here to comment separted cases, like in:
https://github.com/jellyfin/jellyfin-web/blob/24760a1158c9575a2a96a7e65aabac12d217dbb9/src/components/subtitlesync/subtitlesync.js#L143-L146
but I can change it if you prefer.

Reply:
perhaps this?
 `// if showing subtitle sync is enabled && if there is an external subtitle stream enabled`

Reply:
Yeah, I'm not sure this makes to if-statement more readable so I'd rather have it on one line.

Reply:
Ok, no problem.